addons/binding/org.openhab.binding.keba/ESH-INF/thing/kecontact.xml

```diff
@@ -135,5 +136,11 @@
 		<description>Power consumption in Wh. Total energy consumption is added up after each completed charging session</description>
 		<state pattern="%d Wh" readOnly="true"></state>
 	</channel-type>
+    <channel-type id="display" advanced="true">
```

Comment:
It works for me and I guess we will have to solve it in your IDE - this PR should definitely use tabs, no spaces.